code/modules/vehicles/mecha/combat/durand.dm

```diff
@@ -205,13 +205,11 @@ own integrity back to max. Shield is automatically dropped if we run out of powe
 		invisibility = 0
 		flick("shield_raise", src)
 		playsound(src, 'sound/mecha/mech_shield_raise.ogg', 50, FALSE)
-		set_light(l_range = MINIMUM_USEFUL_LIGHT_RANGE	, l_power = 5, l_color = "#00FFFF")
```

Comment:
Related to your lum changes from a previous pr I assume?

Reply:
No, movable lights don't support set_light. They have to be done individually. In this case, we don't need it since the light never changes it has to be on or off.